htdocs/product/fournisseurs.php

```diff
@@ -508,9 +508,9 @@
 				print '<tr><td class="fieldrequired">'.$langs->trans("SupplierRef").'</td><td>';
 				if ($rowid) {
 					print '<input type="hidden" name="ref_fourn_old" value="'.$object->ref_supplier.'">';
-					print '<input class="flat width150" maxlength="30" name="ref_fourn" value="'.$object->ref_supplier.'">';
+					print '<input class="flat width150" name="ref_fourn" value="'.$object->ref_supplier.'">';
```

Comment:
The length in database is 128, so why not changing maxlength="30" into maxlength="128" ?

Reply:
May be somebody needs a longer field and change in DB to varchar(256), and we will have the same problem

Reply:
Another solution will be to get the value of the varchar in DB, and put a maxlength=$value. But is it usefull ?